t_frang/test_http_trailer_split_allowed.py

```diff
@@ -0,0 +1,165 @@
+"""Tests for Frang directive `http_trailer_split_allowed`."""
+from framework import tester
+from helpers import dmesg
+
+
+
+COUNT_WARNINGS_OK = 1
+
+
+ERROR_MSG = 'Frang limits warning is not shown'
+
+RESPONSE_CONTENT = """HTTP/1.1 200 OK\r
+Content-Length: 0\r\n
+Connection: keep-alive\r\n\r\n
+"""
+
+TEMPESTA_CONF = """
+cache 0;
+listen 80;
+
+
+frang_limits {
+    http_trailer_split_allowed true;
+}
```

Comment:
Setting the same header in the headers and trailer parts is uncommon, so I think we should test all the requests for both the `http_trailer_split_allowed` switched on and off.